test/angularODataUtils.spec.ts

```diff
@@ -11,17 +11,25 @@ describe('ODataUtils', () => {
         assert.equal(value, `'test'`);
     });
 
+    it('quoteValue string with quotes', () => {
+        // Act
+        const value: string = ODataUtils.quoteValue('te\'st');
+
+        // Assert
+        assert.equal(value, `'te\'\'st'`);
+    });
+
     it('quoteValue integer', () => {
```

Comment:
Can you please also add a test for boolean?

Like:
`it('quoteValue boolean', () => {`